galleon-pack/layer-metadata-tests/src/test/resources/glow/latest/provisioning-bare-metal.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" ?>
+
+<installation xmlns="urn:jboss:galleon:provisioning:3.0">
+    <feature-pack location="${ee.maven.groupId}:wildfly-galleon-pack:${project.version}"/>
```

Comment:
This should be full.maven.groupId.

Reply:
@bstansberry , good catch! Thank-you, fixed.